index.html

```diff
@@ -393,6 +396,8 @@ <h2 class="form-text">
             id="feedback"
             name="feedback"
             placeholder="Enter text here"
+            required 
+            maxlength="500"
           ></textarea>
           <div class="btn-align">
             <input type="submit" class="btn-3" value="Get in Touch" />
```

Comment:
- [ ] the social icon image is broken kindly fix it.
<img width="158" alt="image" src="https://user-images.githubusercontent.com/77190203/155291989-16cc6767-1bba-45d4-8e62-5fd524276521.png">